dist/operations/update.js

```diff
@@ -35,19 +35,62 @@ class UpdateOperation {
     }
     async execute(input) {
         const ddb = helpers_1.createClient(input.region);
-        const item = input.expressionAttributeValues || await this.read(input.expressionAttributeFiles);
+        let updateExp = ``;
+        let attValues = {};
+        const expressions = await this.buildExpression(updateExp, input);
+        const attributes = await this.buildAttributes(expressions, attValues, input);
+        console.log(expressions);
+        console.log(attributes);
         await ddb.update({
             TableName: input.table,
             Key: input.key,
-            UpdateExpression: `set ${input.updateExpression} = :${input.updateExpression}`,
-            ExpressionAttributeValues: {
-                [`:${input.updateExpression}`]: `${item}`
-            }
+            UpdateExpression: expressions,
+            ExpressionAttributeValues: attributes
         }).promise();
     }
     async read(path) {
         const content = await fs_1.promises.readFile(path, { encoding: "utf8" });
         return JSON.parse(content);
     }
+    async buildExpression(updateExp, input) {
+        const updateExpressions = input.updateExpression.split(',');
+        for (let i = 0; i < updateExpressions.length; i++) {
+            if (i === 0 && updateExpressions.length > 1) {
+                updateExp = 'set'.concat(` ${updateExpressions[i]} = :${updateExpressions[i]},`);
```

Comment:
this is look odd its like mixing programming language conventions.
```
'set'.concat(` ${updateExpressions[i]} = :${updateExpressions[i]},`);
```
is equivelent to
```
`set ${updateExpressions[i]} = :${updateExpressions[i]},`